tests/test_server.py

```diff
@@ -1154,6 +1154,11 @@ def make_source(collector, filename, module, source, req):
     assert len(parameters) == 2
 
 
-def test_server_logs_address(server, caplog):
+@pytest.mark.asyncio(timeout=30)
+async def test_server_logs_address(server, client, caplog):
+    result = await client.create_project("env-test")
+    assert result.code == 200
+    project_id = result.result["project"]["id"]
```

Comment:
just the first call and the 200 code is enough. The other two lines can go.

Reply:
That wasn't enough for the rest server to start, I tried adding something more specific